tests/nightly/test_large_vector.py

```diff
@@ -708,6 +708,111 @@ def test_full():
     assert a[-1] == 3
 
 
+def test_hyperbolic():
+    def test_arccosh(a):
```

Comment:
can this be merged with trignometric functions that are tested in this PR  ? https://github.com/apache/incubator-mxnet/pull/16477

Reply:
can we merge those later ? We divided the work like this

Reply:
this should be quick right ?

Reply:
Done!